### ISSUE TITLE
Open MeTube Optional?

### ISSUE THREAD

**Issue by kushfest** [Rpsl/metube-browser-extension#4]
This is a great extension and super useful for metube users. Any chance you could make it optional for metube to open after using the extension to add a link tok metube? Right now it opens every time and I'd love to be able to choose whether that happens. Thanks for making this!

**Comment by Rpsl** [comment]
as i remember open new tab or change focus to currently opened tab it's techical limitation in work chrome extension. without open tab sometimes download not working correctly.

i not sure but i test it when working on extension. i will try research around this issue but in low priority.

**Comment by kushfest** [comment]
Oh I see. Well that makes sense. It would be cool if possible, but yeah, I get that being a low priority.

**Comment by akashvacher** [comment]
I was able to implement this in my forked repo and it seems to be working well @Rpsl 

I'd be happy to open a PR once #6 is merged as this other branch does depend on the refactoring I did in that PR

Here is the branch if you already have thoughts to share in the meanwhile: https://github.com/akashvacher/metube-browser-extension/tree/silentSend

**Comment by Rpsl** [comment]
#6 was merged. thx
